crates/turbopath/src/lib.rs

```diff
@@ -43,11 +47,18 @@ pub enum PathValidationError {
     #[error("Path is not absolute: {0}")]
     NotAbsolute(PathBuf),
     #[error("Path is not relative: {0}")]
-    NotRelative(PathBuf),
+    NotRelative(String),
     #[error("Path {0} is not parent of {1}")]
     NotParent(String, String),
     #[error("Path {0} is not a unix path")]
     NotUnix(String),
+    #[error("{0} is not a prefix for {1}")]
+    PrefixError(String, String),
+}
+
+pub(crate) fn not_relative_error(bytes: &[u8]) -> PathValidationError {
```

Comment:
We should put this inside an `impl PathValidationError`

Reply:
Done.